src/System.CodeDom/src/System/CodeDom/Compiler/CompilerParameters.evidence.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Nit: Should the file be CompilerParameters.Evidence.cs (upper case "E")? Same for CompilerResults.Evidence.cs?